fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java

```diff
@@ -1326,5 +1331,34 @@ public static ConstantOperator urlExtractParameter(ConstantOperator url, Constan
         }
         return ConstantOperator.createNull(Type.VARCHAR);
     }
+
+    private static void getRecursiveRole(AuthorizationMgr manager, List<String> roleName, Long roleId) {
```

Comment:
This is pretty much like `protected void getAllPredecessorsInner(long roleId, Set<Long> resultSet) throws PrivilegeException `,  how about renaming it to `getAllPredecessorNames()` and put it in `AuthorizationMgr`, it maybe useful in other situations.